jni/Android.mk

```diff
@@ -26,23 +26,25 @@ LOCAL_SRC_FILES := $(TARGET_ARCH_ABI)/lib/libevent_pthreads.a
 include $(PREBUILT_STATIC_LIBRARY)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := libmeasurement_kit
-LOCAL_SRC_FILES := $(TARGET_ARCH_ABI)/lib/libmeasurement_kit.a
-include $(PREBUILT_STATIC_LIBRARY)
-
-include $(CLEAR_VARS)
 LOCAL_MODULE := libssl
 LOCAL_SRC_FILES := $(TARGET_ARCH_ABI)/lib/libssl.a
 include $(PREBUILT_STATIC_LIBRARY)
 
 include $(CLEAR_VARS)
 LOCAL_LDLIBS := -llog -latomic
 # Note to self: the order of libraries matters
-LOCAL_STATIC_LIBRARIES := measurement_kit GeoIP ssl crypto event event_openssl \
+LOCAL_STATIC_LIBRARIES := GeoIP ssl crypto event event_openssl \
```

Comment:
Because the measurement-kit library now is compiled on the fly along with its JNI wrappers whereas before we cross-compiled it, uploaded the binary, and then downloaded it here to cross-compile everything. This is perhaps a good place where to explain my choices:

1. I'd like to stop cross compiling with a custom cross toolchain created by us because using the `ndk-build` command there are less things that can go wrong (in the long run I'd like to ditch also this `ndk-build` approach and use `CMake` instead, since we are already using it to compile for Windows and since now Android studio supports `CMake`).

2. I initially planned to cross compile measurement-kit as before in the usual repository using the `ndk-build` approach, but when I noticed the compiled static libraries where 0.8 GiB (more than 100 MiB compressed) I started thinking whether a less space consuming approach was possible and then I realized I could compile everything here. Now, I believe compiling everything in a single pass is not only good for saving space but also reduces complexity because there are less things than can go wrong. Yet, I realize that this choice has no impact on me but has potential impact on others that now need to recompile the whole measurement-kit.

Do you want me to return to the previous approach where we cross compile the library on the main repository and we publish the cross compiled binaries and here we just add to that the compilation of the JNI wrappers? If you prefer this path, I'll be happy to implement it.